signac/__main__.py

```diff
@@ -724,15 +722,13 @@ def main_config_show(args):
         cfg = config.read_config_file(config.USER_CONFIG_FN)
     else:
         cfg = config.load_config()
-    if cfg is None:
-        if args.local and args.globalcfg:
-            mode = " local or global "
-        elif args.local:
+    if not cfg:
```

Comment:
Both `read_config` and `load_config` are guaranteed to return a `Config` object if they succeed. This change is necessary because `load_config` can return an empty Config if no config files were found. Previously, you would get a printed log (although not an exception) saying that the "project" key was missing as you pointed out [here](https://github.com/glotzerlab/signac/pull/643#discussion_r776492146). That can no longer happen since the key doesn't exist, and we can't replace it with "project_dir" since I plan to remove that from the config as well (in the next PR where I change load_config to return a pair of values). Therefore, the two possibilities are that `cfg is None` -- which means that the user specified `args.local` and the project config didn't exist -- or that `cfg == Config({})`, which can be evaluated as falsy.